portfolio/src/main/java/com/google/sps/servlets/HelloWorld.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.servlets;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+@WebServlet("/MyServelt")
+public class HelloWorld extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<String> contaner = new ArrayList<String>();
```

Comment:
I'll talk about this in our team meeting but you typically assign initialized collections into the broader interfaces, in this case `List<String>`. No need to change, I took a note to talk about this when we all meet.